warp10/src/main/java/io/warp10/standalone/datalog/TCPDatalogConsumer.java

```diff
@@ -127,6 +129,10 @@ public class TCPDatalogConsumer extends Thread implements DatalogConsumer {
 
   private String suffix;
 
+  static {
+    KEEPALIVE_DELAY = Long.parseLong(WarpConfig.getProperty(FileBasedDatalogManager.CONFIG_DATALOG_CONSUMER_KEEPALIVE, TCPDatalogFeederWorker.DEFAULT_KEEPALIVE));
+  }
+
```

Comment:
Can't this class be instantiated before the configuration is set ?  
Remember such a problem in the past...

Reply:
Nope, consumers are created because they are defined in the configuration, so config is always set when this static block is called.